azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/config/AzureComputeHttpApiModule.java

```diff
@@ -46,9 +44,9 @@ protected void bindErrorHandlers() {
    @Override
    protected void installLocations() {
       super.installLocations();
-      bind(ImplicitLocationSupplier.class).
-              to(OnlyLocationOrFirstRegionOptionallyMatchingRegionId.class).
```

Comment:
@nacx I'm having issues in restoring `OnlyLocationOrFirstRegionOptionallyMatchingRegionId` as it fall-back to `providerUri`which is always the same for each region. Wrong supplier?